agents/windows/plugins/mk_mysql.vbs

```diff
@@ -24,7 +24,7 @@ cfg_dir = SHO.ExpandEnvironmentStrings("%MK_CONFDIR%")
 '
 
 Set WMI = GetObject("winmgmts:{impersonationLevel=impersonate}!\\.\root\cimv2")
-Set service_list = WMI.ExecQuery("SELECT * FROM Win32_Service WHERE Name LIKE '%MySQL%' and State = 'Running'")
+Set service_list = WMI.ExecQuery("SELECT * FROM Win32_Service WHERE Name LIKE '%MySQL%' or Name LIKE '%MariaDB' and State = 'Running'")
```

Comment:
You sould use something like `(Name LIKE '%MySQL%' or Name LIKE '%MariaDB')` to make it more obvious that both variants should be running.

Reply:
I do agree totally.